server/server.js

```diff
@@ -453,20 +447,80 @@ app.get("/user/getemployee", function(req, res) {
         console.log(err);
         res.send("Error getting employee from database.");
       } else {
-        console.log("Server 1100: " + rows[0].firstName);
-        var data = rows.map(x => ({
-          id: x.id,
-          firstName: x.firstName,
-          lastName: x.lastName,
-          email: x.email
-        }));
-        //console.log("Server: " + data[0].firstName + " " + data[0].id);
+        var data = rows.map((x) => ({ id: x.id, firstName: x.firstName, lastName: x.lastName, email: x.email }))
+        res.json(data);
+      }
+    }
+  );
+});
+
+app.get("/user/account", function(req, res) {
+  conn.query(
+  "SELECT id, firstName, lastName, email, password FROM user WHERE id=?",
+  [req.query.id],
+    function(err, rows) {
+      if (err) {
+        console.log(err);
+        res.send("Error getting employee from database.");
+      } else {
+        var data = rows.map((x) => ({ id: x.id, firstName: x.firstName, lastName: x.lastName, email: x.email, password: x.password }))
```

Comment:
We likely shouldn't return the password here, and just leave the field blank in the form.

Reply:
thanks for checking. i removed the password.
```suggestion
        var data = rows.map((x) => ({ id: x.id, firstName: x.firstName, lastName: x.lastName, email: x.email }))
```